_pytest/main.py

```diff
@@ -210,7 +210,9 @@ def __getitem__(self, key):
             return self.parent.keywords[key]
 
     def __setitem__(self, key, value):
-        self._markers[key] = value
+        from _pytest.mark import MarkDecorator
+        if not isinstance(self._markers.get(key), MarkDecorator):
```

Comment:
no, its currently not fixable due to broken internal (i forsee any minimal attempt causing more regressions)